src/System.Text.Http.Parser/HttpMethod.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace System.Text.Http.Parser
+{
+    public enum HttpMethod: byte
```

Comment:
0 not that it matters as we don't expose these to user code. We call it pubternal.